src/components/AppSection/AppSection.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useContext, useEffect, useState } from 'react';
+import cn from 'classnames';
+
+import { PostDetails } from '../PostDetails';
+import { PostsList } from '../PostsList';
+import { Loader } from '../Loader';
+import { UserSelector } from '../UserSelector';
+import { AppContext } from '../AppContext';
+import * as getService from '../../services/AppServices';
+
+export const AppSection: React.FC = () => {
+  const {
+    setUsers,
+    selectedUser,
+    userPosts,
+    setUserPosts,
+    selectedPost,
+  } = useContext(AppContext);
+  const [loading, setLoading] = useState(false);
```

Comment:
Rename all boolean variables using [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```